articles/quickstart/spa/react/interactive.md

```diff
@@ -28,19 +31,34 @@ A callback URL is a URL in your application where Auth0 redirects the user after
 
 A logout URL is a URL in your application that Auth0 can return to after the user has been logged out of the authorization server. This is specified in the returnTo query parameter. If this field is not set, users will be unable to log out from the application and will get an error.
 
-## Install the Auth0 React SDK {{{ data-action=code data-code="app.js#5:10" }}}
+### Configure Allowed Web Origins
+
+You need to add the URL for your app to the Allowed Web Origins field in your Application Settings. If you don't register your application URL here, the application will be unable to silently refresh the authentication tokens and your users will be logged out the next time they visit the application, or refresh the page. 
+
+<!-- TODO this normally comes in from the _getting_started include (along with other things). Should this be part of app configuration? Any logged in/logged out different behaviors needed? -->
+::: note
+If you are following along with the sample project you downloaded from the top of this page, you should set the **Allowed Web Origins** to `http://localhost:3000`.
+:::
+
+## Install the Auth0 React SDK {{{ data-action=code data-code="index.js#13:22" }}}
 
 In your terminal, run the command within your application directory to install the Auth0 React SDK.
 
+<!-- TODO the code block component includes a feedback block. Do we want that? If so, need to differentiate the feedback for this new QS design -->
```

Comment:
can you give a visual of this

Reply:
<img width="729" alt="Screen Shot 2021-11-17 at 2 12 12 PM" src="https://user-images.githubusercontent.com/276225/142275092-ad3c7f6a-c39f-4dba-b83b-2a19687422bd.png">